www/sass/modules/_base.scss

```diff
@@ -12,7 +12,8 @@
 $color-white: #fff; // white
 
 // status colors and text
-$state-colors: 
+$state-colors:
+beginning #54FF2D $color-white "Beginning",
```

Comment:
When is the state "beginning"?

Waiting for an available slave? Running?

Reply:
Probably wait for a slave. I use label which I found in `buildbot/status/results.py`
BEGINNING = -1